docs/journal/01-cold-storage.md

```diff
@@ -1,74 +1,90 @@
-# Journal Entry #1
-_Capturing event data and saving it in its raw format to cold storage_
+# Journal 1: Event Ingestion and Cold Storage
```

Comment:
I'd like to refer to it as "Long Term" rather than "Cold", at least when we are introducing the topic.